src/pages/guides/uxp_guide/uxp-misc/manifest-v4/index.md

```diff
@@ -86,9 +86,21 @@ Key | Type | Description
 `theme` | `string[]` | Array of themes this icon supports. Photoshop supports `lightest`, `light`, `dark`, and `darkest`. If all themes are compatible with the icon, you can use `all`. (Default is `all`).
 `species` | `string[]` | Identifies the type of icon and where it would make sense to display it. The default is `generic`, meaning that Photoshop is free to use this icon anywhere.
 
+## Hosts
+
+The `host` field is an _array_ of objects matching the `HostDefinition` format specified below. These entries allow your plugin to be ran on multiple apps such as Adobe XD or Photoshop. Optionally, the field can contain a HostDefinition instead of a full array if only one type of app is being supported.
```

Comment:
Let's focus on the single HostDefinition case and then indicate that during development, it may be useful to supply an array of HostDefinitions. (At submission time, the plugin bundle supplied has to only have one host definition.)

Reply:
Sounds good! Changed in my most recent commit